bin/drk/src/main.rs

```diff
@@ -48,6 +48,8 @@ use darkfi::{
     },
 };
 
+const MAX_DECIMAL_PLACES: usize = 8;
```

Comment:
aggstam: I've made the updates, also included a title above the const declaration "/// Constant declaration" to keep it in the same format as the module titles. Let me know if you want it removed though

Reply:
These are not titles, they are documentation comments, describing what each module is, which will then be rendered in crate docs page.
Either remove it or update its content with something meaningfull describing what the constant is doing, like in these [examples](https://github.com/darkrenaissance/darkfi/blob/master/src/validator/pow.rs#L48-L75).

After a more thorough review, the constant should be moved in `bin/drk/src/wallet.rs`, since its a balance thing of the wallet, not the main `drk` functionality. Additionally, I see that not all hardcoded values have been replaced, like [here](https://github.com/darkrenaissance/darkfi/blob/master/bin/drk/src/wallet_dao.rs#L84), and decode_base10() should also use same constant, like [here](https://github.com/darkrenaissance/darkfi/blob/master/bin/drk/src/main.rs#L903). All instances of encode/decode in `drk` must use same constant.

Reply:
Thanks for the feedback. I did ask about those other hardcoded values in dev chat but wasn't sure because they didn't have a 'to do' or 'fix me' comment. Will sort it out.